pkgs/unified_analytics/lib/src/enums.dart

```diff
@@ -153,6 +153,16 @@ enum DashEvent {
     required this.description,
     this.toolOwner,
   });
+
+  /// This takes in the string label for a given [DashEvent] and returns the
+  /// enum for that string label.
+  static DashEvent? getDashEventByLabel(String label) {
```

Comment:
nit: rename to `byLabel` or `fromLabel` so that it reads `DashEvent.fromLabel` when using this method.

also, starting method names with "get" is not recommended by effective dart: https://dart.dev/effective-dart/design#avoid-starting-a-method-name-with-get